CMakeLists.txt

```diff
@@ -36,6 +36,21 @@ if (NOT glm_POPULATED)
     )
 endif()
 
+option(BUILD_DOCUMENTATION "Build the HTML documentation" ON)
```

Comment:
I think it would be best if the documentation is not built by default. It will put Doxygen as an optional dependency, instead of a required one.